src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+@import "../utils/variables";
+
+.nav {
+  height: 100%;
+
+  &__list {
+    display: flex;
+    height: 100%;
+    margin: 0;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 100%;
+
+    color: $main-color;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__link:hover {
```

Comment:
This hover move to link 
![image](https://user-images.githubusercontent.com/70009205/227006605-c746d205-4fd9-4ee6-8544-526d9901ba7d.png)